changelogs/CHANGELOG.rst

```diff
@@ -4,6 +4,29 @@ Cisco Ios Collection Release Notes
 
 .. contents:: Topics
 
+v1.0.2
+======
+
+Release Summary
+---------------
+
+- Re-releasing 1.0.2 with updated changelog.
+
+v1.0.1
+======
+
+Minor Changes
+-------------
+
+- Add changelogs for 1.0.0 release (https://github.com/ansible-collections/cisco.ios/pull/90)
```

Comment:
The entry is not required. This got generated because the changelog PR for v1.0.0 had a changelog entry for itself.